cart/views.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from django.views import View
+from django.http import JsonResponse
+
+from user.utils import LoginConfirm
+from .models import Cart
+from task.models import Product
+from user.models import User
+
+class CartAdd(View):
+    @LoginConfirm
+    def post(self, request):
+        data = json.loads(request.body)
+        info = Product.objects.all()
+        try:
+            p_id = Product.objects.get(item=data['item']).id
+            user = User.objects.get(id = request.user.id)
+            if Cart.objects.filter(item=data['item'], user_id=request.user.id).exists():
+                return JsonResponse({'message':'이미 담겼어요'})
+            else:
+                Cart(item      = data['item'],
+                    count      = data['count'],
+                    product_id = p_id,
+                    user_id    = user.id
+                ).save()
+
+                return JsonResponse({'message':'Added'})
+        
```

Comment:
```suggestion
            
            Cart(item      = data['item'],
                count      = data['count'],
                product_id = p_id,
                user_id    = user.id
            ).save()

            return JsonResponse({'message':'Added'})
        
```

해당 코드는 위에서 if 문을 만족시키지못할 때 바로 도달 할 수 있는 부분이므로 `else`문이 없어도 동작합니다

Reply:
else 지우고 indent 조정하였습니다.